.github/bin/common.sh

```diff
@@ -69,6 +69,10 @@ initStableTemplates() {
   export LAST_PACKAGE_VERSION
   PREVIOUS_PACKAGE_VERSION=$(echo "${versions[@]}" | jq '.data.repository.refs.edges[0].node.name | sub("\""; "")' | tr -d '"')
   export PREVIOUS_PACKAGE_VERSION
+}
+
+initStableTemplates() {
```

Comment:
`initStableTemplates` is used by other scripts.
I am afraid if `LAST_PACKAGE_VERSION` and `PREVIOUS_PACKAGE_VERSION` will be undefined there.
Can we revert this and refactor next time?

Reply:
I'm not renaming this function. The same function still exists. I'm just creating a new function which get the Che versions from GH.